.github/workflows/prod_deploy.yml

```diff
@@ -0,0 +1,75 @@
+name: Prod Deployment
+
+on:
+  workflow_run:
+    workflows: ["Dev Deployment"]
+    types:
+      - completed
+env:
+  IMAGE_TAG: ${{ github.sha }}
+
+jobs:
+  deploy:
+    name: Deploy infrastructure and chart
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: runs-on
+    permissions:
+      id-token: write
+      contents: read
+    environment:
+      name: prod
+    steps:
+      - name: Sleep for 1 minute
+        run: echo "Waiting for 60 seconds to allow for abort of prod deployment" && sleep 60
+        shell: bash
+
+      - uses: actions/checkout@v4
+
+      - uses: citizensadvice/python-poetry-setup-action@v1
+        with:
+          install_cdk: true
+          project_root: "./cdk"
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: "arn:aws:iam::${{ vars.AWS_ACCOUNT }}:role/LocalOfficeSearchApiDeployment"
+          role-session-name: local-office-search-api-deploy-prod-workflow
+          aws-region: eu-west-1
+
+      - name: Deploy CDK
+        run: |
+          set -eo pipefail
+          cd ./cdk
+          poetry run cdk deploy "prod/**" LocalOfficeSearchApiEcrRepo --concurrency 2 --ci --require-approval never
```

Comment:
Have you noticed any difference when adding the `--ci` flag? I see it [changes the log output](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk/README.md#running-in-ci) from `stderr` to `stdout`. Just wondering if we should go back and add it to our other repos?

Reply:
I'm honestly not sure why this is set, I've probably copied it from somewhere else and just assumed it'd be a good idea

Reply:
It might be a good idea! 😃 I'm just not familiar with it.